src/pybind/mgr/dashboard/frontend/e2e/user-mgmt.po.ts

```diff
@@ -0,0 +1,118 @@
+import { $, browser, by, element } from 'protractor';
+import { Helper } from './helper.po';
+import { PageHelper } from './page-helper.po';
+
+export class UserMgmtPageHelper extends PageHelper {
+  pages = {
+    index: '/#/user-management',
+    users: '/#/user-management/users',
+    userscreate: '/#/user-management/users/create',
+    roles: '/#/user-management/roles',
+    rolescreate: '/#/user-management/roles/create'
+  };
+
+  userCreate(username, password, name, email) {
+    this.navigateTo('userscreate');
+
+    // fill in fields
+    element(by.id('username')).sendKeys(username);
+    element(by.id('password')).sendKeys(password);
+    element(by.id('confirmpassword')).sendKeys(password);
+    element(by.id('name')).sendKeys(name);
+    element(by.id('email')).sendKeys(email);
+
+    // Click the create button and wait for user to be made
+    const createButton = element(by.cssContainingText('button', 'Create User'));
+    this.moveClick(createButton).then(() => {
+      browser.wait(Helper.EC.presenceOf(this.getTableCell(username)), 10000);
```

Comment:
Same for other occurrences. 
```suggestion
      browser.wait(Helper.EC.presenceOf(this.getTableCell(username)), Helper.TIMEOUT);
```